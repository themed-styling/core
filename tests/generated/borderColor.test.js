import 'jest-styled-components'

import { borderColor } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(borderColor, 'borderColor')
