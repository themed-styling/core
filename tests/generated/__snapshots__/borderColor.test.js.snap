// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`borderColor renders 0 1`] = `
Array [
  .c0 {
  border-color: #0;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: #0;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: #0;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: #0;
}

<div
    className="c0"
  />,
]
`;

exports[`borderColor renders function 1`] = `
Array [
  .c0 {
  border-color: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: has;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: has;
}

<div
    className="c0"
  />,
]
`;

exports[`borderColor renders invalid theme path 1`] = `
Array [
  .c0 {
  border-color: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`borderColor renders negative number 1`] = `
Array [
  .c0 {
  border-color: #-54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: #-54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: #-42365;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: #-42365;
}

<div
    className="c0"
  />,
]
`;

exports[`borderColor renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    border-color: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-color: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-color: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-color: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    border-color: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-color: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-color: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-color: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-color: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-color: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-color: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-color: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-color: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-color: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-color: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-color: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderColor renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    border-color: July;
  }
}

@media April {
  .c0 {
    border-color: August;
  }
}

@media January {
  .c0 {
    border-color: September;
  }
}

@media April {
  .c0 {
    border-color: November;
  }
}

@media June {
  .c0 {
    border-color: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    border-color: July;
  }
}

@media April {
  .c0 {
    border-color: August;
  }
}

@media January {
  .c0 {
    border-color: September;
  }
}

@media April {
  .c0 {
    border-color: November;
  }
}

@media June {
  .c0 {
    border-color: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-color: January;
  }
}

@media April {
  .c0 {
    border-color: December;
  }
}

@media January {
  .c0 {
    border-color: April;
  }
}

@media April {
  .c0 {
    border-color: May;
  }
}

@media June {
  .c0 {
    border-color: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-color: January;
  }
}

@media April {
  .c0 {
    border-color: December;
  }
}

@media January {
  .c0 {
    border-color: April;
  }
}

@media April {
  .c0 {
    border-color: May;
  }
}

@media June {
  .c0 {
    border-color: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderColor renders postitive number 1`] = `
Array [
  .c0 {
  border-color: #54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: #54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: #42365;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: #42365;
}

<div
    className="c0"
  />,
]
`;

exports[`borderColor renders string 1`] = `
Array [
  .c0 {
  border-color: July;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: July;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: June;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: June;
}

<div
    className="c0"
  />,
]
`;

exports[`borderColor renders valid theme path 1`] = `
Array [
  .c0 {
  border-color: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-color: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
